### PR TITLE
pred bracers and decloaking

### PR DESCRIPTION
**Pull request by stanalbatross**
## About The Pull Request

le bugge fixe

## Changelog

:cl:
fix: pred bracers will now decloak you when you drop them
fix: pred bracers will now decloak you when you put them into bags
fix: pred bracers will now properly decloak you before shocking you if they run out of charge
fix: if you get shocked while trying to activate something with a pred bracer, it won't do the action you were trying to do
fix: also fixed dropship autolaunch delay having a broken input 
/:cl:

### CODE REVIEW

File: code/modules/cm_preds/yaut_bracers.dm [hunk 1]
```diff
@@ -95,7 +95,13 @@
 	//Non-Yautja have a chance to get stunned with each power drain
 	if(!HAS_TRAIT(M, TRAIT_YAUTJA_TECH) && !M.hunter_data.thralled)
 		if(prob(15))
+			if(istype(src, /obj/item/clothing/gloves/yautja/hunter))
+				var/obj/item/clothing/gloves/yautja/hunter/huntergloves = src
+				if(huntergloves.cloaked)
+					huntergloves.decloak(M)
```
Comment (morrowwolf) [inline code/modules/cm_preds/yaut_bracers.dm:101]: Variable readability for "M"

File: code/modules/cm_preds/yaut_bracers.dm [hunk 2]
```diff
@@ -273,15 +279,24 @@
 		//Non-Yautja have a chance to get stunned with each power drain
 		if(!isYautja(H))
 			if(prob(15))
+				decloak(H)
```
Comment (morrowwolf) [inline code/modules/cm_preds/yaut_bracers.dm:282]: Variable readability for "H"

File: code/modules/cm_preds/yaut_bracers.dm [hunk 3]
```diff
@@ -273,15 +279,24 @@
...
 	move_chip_to_bracer()
+	if(cloaked)
+		decloak(user)
 	..()
 
+/obj/item/clothing/gloves/yautja/hunter/on_enter_storage(obj/item/storage/S)
+	if(ishuman(loc))
+		var/mob/living/carbon/human/H = loc
+		if(cloaked)
+			decloak(H)
```
Comment (morrowwolf) [inline code/modules/cm_preds/yaut_bracers.dm:297]: Variable readability for "H"

**Review by morrowwolf — CHANGES_REQUESTED** [review]

File: code/modules/cm_preds/yaut_bracers.dm [hunk 4]
```diff
@@ -95,7 +95,13 @@
 	//Non-Yautja have a chance to get stunned with each power drain
 	if(!HAS_TRAIT(M, TRAIT_YAUTJA_TECH) && !M.hunter_data.thralled)
 		if(prob(15))
+			if(istype(src, /obj/item/clothing/gloves/yautja/hunter))
```
Comment (morrowwolf) [inline code/modules/cm_preds/yaut_bracers.dm:98]: I may be reading this wrong but should this be set up via overriding in the child type?

**Review by morrowwolf — CHANGES_REQUESTED** [review]

File: code/modules/cm_preds/yaut_bracers.dm (hunk 4 above)
Comment (stanalbatross) [reply code/modules/cm_preds/yaut_bracers.dm:98]: I have to do it like this to execute it before the shock_user() or they will drop the gloves in shock_user() and it won't work

also don't want to dupe code

Comment (stanalbatross) [reply code/modules/cm_preds/yaut_bracers.dm:100]: nvm can do better

File: code/modules/shuttles/shuttle_console.dm [hunk 5]
```diff
@@ -490,7 +490,7 @@ GLOBAL_LIST_EMPTY(shuttle_controls)
 			if(!dropship.automated_launch) //If we're toggling it on...
 				var/auto_delay
 				auto_delay = tgui_input_number(usr, "Set the delay for automated departure after recharging", "Automated Departure Settings", DROPSHIP_MIN_AUTO_DELAY, DROPSHIP_MAX_AUTO_DELAY, DROPSHIP_MIN_AUTO_DELAY)
-				dropship.automated_launch_delay = Clamp(auto_delay SECONDS, DROPSHIP_MIN_AUTO_DELAY, DROPSHIP_MAX_AUTO_DELAY)
+				dropship.automated_launch_delay = Clamp(auto_delay, DROPSHIP_MIN_AUTO_DELAY, DROPSHIP_MAX_AUTO_DELAY)
```
Comment (morrowwolf) [inline code/modules/shuttles/shuttle_console.dm:493]: This is currently off by a factor of 10 when inputting. The intent would be to input seconds but I'm inputting deciseconds.

**Review by morrowwolf — CHANGES_REQUESTED** [review]
Yautja stuff looks good. Just this last nitpick and good to go.

**Review by morrowwolf — APPROVED** [review]
Code LGTM

**Pull request merged**